public/500.html

```diff
@@ -1,66 +1,62 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>We're sorry, but something went wrong (500)</title>
+  <title>Something went wrong (500)</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  body {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
-
-  div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
-
-  div.dialog > div {
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #BBB;
-    border-top: #B00100 solid 4px;
-    border-top-left-radius: 9px;
-    border-top-right-radius: 9px;
-    background-color: white;
-    padding: 7px 12% 0;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
+<meta content="IE=edge" http-equiv="X-UA-Compatible">
+<meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+<meta content="NONE,NOARCHIVE" name="robots">
+<meta name="csrf-param" content="authenticity_token">
+<meta name="csrf-token" content="7CZFCDlIOrVe/vFDj8hml6BwwkbdI9ifYiGbxlikx9jo1cH10ovX3sJGuqSoLFq/D8bLsTa4hRLtAdlqNEObmQ==">
+<link rel="stylesheet" media="all" href="/assets/rails_admin/rails_admin-aa8fcab07991cfc251160f921e9001fd.css?body=1">
+<script src="/assets/jquery-2ebc4d7dee7ed5002a2ec81cda5c42ea.js?body=1"></script>
+<script src="/assets/rails_admin/bootstrap/bootstrap-transition-349e2c80e43210f20936b0c48cb0322f.js?body=1"></script>
```

Comment:
Do we regenerate these error files every time we deploy? The URL has the hash fragment for the asset which doesn't get updated unless the task `asset:precompile` is run before we deploy. That is the reason why the CSS is included in the HTML markup in the older version. How are we ensuring that these get updated for every deployment?